dashboard/app/views/pd/regional_partner_contact_mailer/matched.html.haml

```diff
@@ -1,17 +1,33 @@
-%p
-  Hello
-  = "#{@name}" + ','
+%p= "Hello #{@name},"
 
 %p
-  The following person has requested to connect with their local Code.org Regional Partner
-  = link_to('through this form', 'https://studio.code.org/pd/regional_partner_contact/new')
-  and we've identified that they are in your region. Please follow up with them directly at your convenience.
+  A teacher or administrator in your region has asked us to connect them with their local Code.org
+  Regional Partner. We let them know that you’ll get back to them. Please follow up with them
+  directly at your convenience.
 
-= render partial: 'form_body'
+%div{style: 'margin-left: 1em;'}
+  = render partial: 'form_body'
+
+%p
+  %strong Why is this person contacting you?
+%p
+  We encourage administrators and teachers to fill out
+  = link_to 'this form', CDO.studio_url('/pd/regional_partner_contact/new', CDO.default_scheme)
+  if they want to learn more about
+  expanding CS in their schools. You can reach out to them to let them know about your resources
+  and ask how you can help them do this.
+%p
+  We also ask educators to fill out the form to find out when applications open in their area.
+  You should add their email to your list to let them know when you open applications for 6-12 PD
+  or when you set up a K-5 workshop they can join.
+%p
+  They may also use this form to ask you a question about the Professional Learning Program.
 
 %p
   Best,
   %br
-  Tanya Parker
+  &nbsp;
+  %br
+  Anthonette Pena
```

Comment:
Nit: (we had to do this in teacher application work) but Pena should be Peña

Reply:
Thanks!  We need to update https://code.org/about/team

Reply:
I don't think we can because the person's name has to match the image file name and we've had problem with accents in file names in the past.

Reply:
Thanks @tanyaparker, I can note this down but ultimately I don't think any changes to the content pipeline will be needed to eliminate this requirement on this page -- there just needs to be another column in the same gsheet which specifies the filename.